model/task/expected_duration.go

```diff
@@ -92,21 +95,3 @@ func getExpectedDurationsForWindow(name, project, buildvariant string, start, en
 
 	return results, nil
 }
-
```

Comment:
Can we add in the PR description that this was removed only because it was unused

Reply:
comments also provide history so this thread covers that as well i think